packages/ra-ui-materialui/src/layout/AppBar.tsx

```diff
@@ -147,17 +150,19 @@ const AppBar = props => {
     );
 };
 
+interface Props {
```

Comment:
Hello @josephktcheung can you please add `title` property as `TitleComponent` from `ra-core`

Reply:
Hi @TheHyphen thanks for the review. Good catch on this, from what I understand `AppBar` no longer uses `title` prop, the reason why it's not removed is because [we inject `title` prop to `AppBar` in `Layout`](https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/layout/Layout.js#L124). If we don't de-structure `AppBar` props [here](https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/layout/AppBar.js#L90), we may pass a wrong `title` type to material-ui's `AppBar` by spreading the `rest` prop [here](https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/layout/AppBar.js#L101)

Therefore, I question the need of passing `title` prop when we `cloneElement` in https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/layout/Layout.js#L124. I think `title` prop can be removed from `AppBar` which we did previously https://github.com/marmelab/react-admin/commit/a9e4e0d8b3a063eb686e4a15bbae94384a1c1528 but somehow reverted back by @fzaninotto https://github.com/marmelab/react-admin/commit/0f1b4043a37af1c15579be93a18474fdbc2c7a0d#diff-bbb6c2d0f8bfb7588ebadf3e64ed2e57.

@fzaninotto please confirm if `title` prop is still relevant for `AppBar` or is it a legacy code?